src/Routing/UrlGenerator.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Laravel\Lumen\Routing;
 
-use Laravel\Lumen\Application;
+use Illuminate\Contracts\Foundation\Application;
```

Comment:
@taylorotwell told me to send the pull to the master branch, I did push it to 5.1 before. So... what should I do now?